src/components/menu-bar/menu-bar.css

```diff
@@ -52,16 +52,28 @@
 .language-icon {
     height:  1.5rem;
     vertical-align: middle;
+    margin: 0 .125rem 0 0;
 }
 
-.language-caret {
-    margin: 0 .125rem;
+[dir="rtl"] .language-icon {
+    margin: 0 0 0 .125rem;
```

Comment:
I wish we could use `margin-inline-[start|end] instead. But we need to support browser versions [without support](https://caniuse.com/?search=margin-inline).

https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline